doc/changelog/11-infrastructure-and-dependencies/11618-loadpath+split_ml_handling.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
```

Comment:
It seems to me that this should be "Removed" since all the changelog entry is about removal.

Reply:
I used changed as we changed the semantics of includes, I dunno, I see this more as a change on how loadpaths are handled [same now than in OCaml] vs a real removal, maybe we could add both?

Reply:
It's fine to keep Changed if you think this is more appropriate.